src/modules/transaction/entity/transaction.database-model.ts

```diff
@@ -0,0 +1,12 @@
+import { TransactionTypeEnum } from './transaction.type.enum';
+
+export class TransactionDatabaseModel {
+    id?: number;
+    user_id?: string;
```

Comment:
@PCalape Why are these optional?

Reply:
Hello @arnellebalane my bad on this one, this was due to a refactoring move in which I did not continue. This should be required, will change this one. Thanks!

Reply:
Done updating this one @arnellebalane thanks!